redismodule.h

```diff
@@ -1,405 +0,0 @@
-#ifndef REDISMODULE_H
```

Comment:
Why is all of this file deleted?

Reply:
Lab 8 gave me the idea to do this because redismodule.h is a header file provided by the RedisModuleSDK tool kit (directory) that RedisLabs provides and is necessary to include in all redis module files. This gives me either the option of just copy and pasting the header file into our directory (which is what I did first) or adding a submodule like we did in Lab 8 so that whenever RedisLabs decides to update the redismodule.h file, no one has to manually update the one in our repository. @valeriehan6